src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,671 @@
+# SOFTWARE, OR ANY WARRANTY THAT THE SOFTWARE WILL BE ERROR FREE. IN NO EVENT
+# SHALL NIST BE LIABLE FOR ANY DAMAGES, INCLUDING, BUT NOT LIMITED TO, DIRECT,
+# INDIRECT, SPECIAL OR CONSEQUENTIAL DAMAGES, ARISING OUT OF, RESULTING FROM,
+# OR IN ANY WAY CONNECTED WITH THIS SOFTWARE, WHETHER OR NOT BASED UPON
+# WARRANTY, CONTRACT, TORT, OR OTHERWISE, WHETHER OR NOT INJURY WAS SUSTAINED
+# BY PERSONS OR PROPERTY OR OTHERWISE, AND WHETHER OR NOT LOSS WAS SUSTAINED
+# FROM, OR AROSE OUT OF THE RESULTS OF, OR USE OF, THE SOFTWARE OR SERVICES
+# PROVIDED HEREUNDER.
+
+# Distributions of NIST software should also include copyright and licensing
+# statements of any third-party software that are legally bundled with the
+# code in compliance with the conditions of those licenses.
+
+from datetime import datetime
+import json
+import os
+import sas
+import sas_testcase
+from reference_models.ppa_model.ppa_ref_model import ppa_creation_model_byList
+from util import winnforum_testcase, configurable_testcase, writeConfig, loadConfig
+from sas import _RequestGet
+from shapely.geometry import Polygon
+
+class PPACreationTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+
+  def generate_PCR_1_default_config(self, filename):
+    # Step 1: Load PAL Database Record
+    pal_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+    pal_record['channelAssign'] = {
+      "primaryAssignmen": {
+        "lowFrequency": 3550000000,
+        "highFrequency": 3560000000
+      },
+      "secondaryAssignment": {
+        "lowFrequency": 3580000000,
+        "highFrequency": 3590000000
+      }
+    }
+    pal_record['userId'] = "pal_user"
+    pal_record['palId'] = "pal/10-2017/123456789/A"
+
+    # Step 2: Load the device_a in DP Test Harness
+    device_a = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_a['installationParam']['latitude'] = 39.0119
+    device_a['installationParam']['longitude'] = -98.4842
+    device_a['userId'] = pal_record['licensee']
+
+    # Step 2: Load the device_b in DP Test Harness
+    device_b = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_b['installationParam']['latitude'] = 39.139836
+    device_b['installationParam']['longitude'] = -97.770094
+    device_b['userId'] = pal_record['licensee']
+
+    url = 'sas_testUUT_url will be provided in the future'
+
+    config = {
+        'url': url,
+        'device_a': device_a,
+        'device_b': device_b,
+        'pal_record': pal_record
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_PCR_1_default_config)
+  def test_WINNF_FT_S_PCR_1(self, config_filename):
+    # Load the Config file
+    config = loadConfig(config_filename)
+    url = config['url']
```

Comment:
Is there value to pulling these out? It seems valuable to keep track of what's coming from a config.

Reply:
Do you mean I do not need to define variable like "url"? I should directly use config['url']?

Reply:
url is   assigned with value as sas_admin URL